### DISCUSSION TITLE
为啥没有主星之外的星曜亮度信息呢？？

### DISCUSSION THREAD

**Discussion by rainteen** [SylarLong/iztro#208] [Q&A]
为啥没有主星之外的星曜亮度信息呢？？
三合派的星曜有100多颗，请问有这些星曜的亮度信息吗？？

**Comment by SylarLong** [comment]
目前有资料的星耀亮度都已经加上去了，如果师兄有更完整的星耀亮度信息又可以提供的话，还请分享出来为开源做一份贡献。如果不方便分享又想使用 iztro 也是可以的，iztro提供了全局配置，可以自行配置星耀亮度。

[参考连接](https://ziwei.pro/posts/config-n-plugin.html)
